kivy/tests/test_environ_cli.py

```diff
@@ -0,0 +1,49 @@
+# coding=utf-8
+
+from os import environ
+import shlex
+import subprocess
+import sys
+
+import pytest
+
+ENV_NAME = "KIVY_NO_ARGS"
+KIVY_ENVS_TO_EXCLUDE = ("KIVY_UNITTEST", "KIVY_PACKAGING")
+
+EXPECTED_STR = "Kivy Usage"
+
+TRUTHY = {"true", "1", "yes"}
+FALSY = {"false", "0", "no", "anything-else"}
+
+SAMPLE_VALUES = {*TRUTHY, *FALSY}
+
+
+def _patch_env(*filtered_keys, **kw):
+    env = {k: v for k, v in environ.items() if k not in filtered_keys}
+    env.update(kw)
+    return env
+
+
+def _kivy_subproces_import(env):
+    return subprocess.check_output(
```

Comment:
It's failing because kivy returns a non-zero value (https://github.com/kivy/kivy/blob/master/kivy/__init__.py#L372). So you need to disable checking the error return value.

Reply:
roger